kube/helm/carbonj/values.yaml

```diff
@@ -0,0 +1,32 @@
+carbonj:
+  shard:
+    id: 1
+    volumeId: vol-067d966ecbef3123a
```

Comment:
This is just a sample value?

Reply:
yeah, since the volumes are being created dynamically I think I will remove it